router/batchrouter/batchrouter.go

```diff
@@ -1396,6 +1396,7 @@ func (brt *HandleT) GetWorkspaceIDForDestID(destID string) string {
 }
 
 func (brt *HandleT) setMultipleJobStatus(asyncOutput asyncdestinationmanager.AsyncUploadOutput, rsourcesStats rsources.StatsCollector) {
+	jobParameters := []byte(fmt.Sprintf(`{"destination_id": %q}`, asyncOutput.DestinationID)) // TODO: there should be a consistent way of finding the actual job parameters
```

Comment:
@cisse21 for async batch destinations we are missing the original job parameters, or at least I wasn't able to figure out how to find them. The current workaround will serve us until the need comes to include an additional filtering parameter in batchrouter.